modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/DefaultCodegen.java

```diff
@@ -904,6 +904,10 @@ public CodegenParameter fromParameter(Parameter param, Set<String> imports) {
       p.required = param.getRequired();
     p.jsonSchema = Json.pretty(param);
 
+    if(param instanceof QueryParameter) {
+        p.defaultValue = ((QueryParameter)param).getDefaultValue();
+    }
+
```

Comment:
Is it correct to say that only the default value of query parameters (but not form parameters nor header parameters) is supported ?

Reply:
What about in the Models? I get defaults for some types (boolean) but not for others (integer).